src/main/java/uk/gov/digital/ho/hocs/casework/api/StageResource.java

```diff
@@ -58,13 +58,6 @@ ResponseEntity<CreateStageResponse> createStage(@PathVariable UUID caseUUID, @Re
         return ResponseEntity.ok(CreateStageResponse.from(stage));
     }
 
-    @Authorised(accessLevel = AccessLevel.READ)
```

Comment:
Can this be deprecated rather than removed for now, removing endpoints are `Major` changes. But a deprecation will allow this to be updated without breaking the contract with Workflow.

Reply:
done